### DISCUSSION TITLE
AES

### DISCUSSION THREAD

**Discussion by dancerqueen** [dyne/tomb#518] [Q&A] [answered]
What AES is used?  How do i see what AES is used?

Reed web page looks like link to https://www.iso.org/standard/54531.html maybe means its 128 but i dont know how to see for sure.  I know how to do gpg --list-packets for gpg but it dont work for me with tomb file.  How do u use command here to find out for tomb file?

I dont know if web page is out of date and 256 is used now or if tomb doesnt use 256.  Many thanks to anybody that help me!

**Comment by roddhjav** [comment, marked as answer]
AES is used by cryptsetup, not gpg, by default it is AES 256 with the XTS mode:

https://github.com/dyne/Tomb/blob/0d06c994cf09cd89c1a5eefd027b8bc421fa5e4e/tomb#L2132

**Reply by dancerqueen** [reply]
Thanks u 4 telling me this, Im not so smart with this stuff but I trying to learn & this help me.  What is command to see what encrytion was used for a tomb file?  This helps me learn & i promise no question after this!!  Thanks you again 4 help me!

**Reply by roddhjav** [reply]
This one (open the file to see it in its context): 

https://github.com/dyne/Tomb/blob/0d06c994cf09cd89c1a5eefd027b8bc421fa5e4e/tomb#L2225

**Reply by dancerqueen** [reply]
Thanks u again 4 help me!!  No questions added bcause i promise b 4 in last msg.  This help me know where to start to learn about this.  I try to use this but it dont work 4 me cause i dont know what tombloop is and what to use for ciper so i will try 2 learn this now but thanks u again 4 help me with how 2 start 2 learn, i didnt know what 2 do b 4 you help me.
